.circleci/config.yml

```diff
@@ -84,7 +126,7 @@ references:
 
   container_config: &container_config
     docker:
-      - image: devdemisto/content-build:3.0.0.33772  # disable-secrets-detection
+      - image: cimg/python:3.10-node # disable-secrets-detection
```

Comment:
we should use our images.
if you want to use another image, please open a PR in the dockerfiles repo.

Reply:
Why should we use our images?
The recommendation for CircleCI is to use the official CircleCI images.
The dependencies on top of this image could be downloaded with CircleCI official orb tools, with their cache.

Reply:
Edited the content-build image to be based on `cimg/python:3.10-node`